artemis-server/src/main/java/org/apache/activemq/artemis/core/config/amqpBrokerConnectivity/AMQPMirrorBrokerConnectionElement.java

```diff
@@ -75,6 +77,12 @@ public AMQPMirrorBrokerConnectionElement setQueueCreation(boolean queueCreation)
       return this;
    }
 
+   public boolean isForward () { return forward; }
```

Comment:
I'd rather this use the same formatting as other accessors and not jam it all on one line, and also space things as others are done here.

Reply:
Same. Its not allowed by the checkstyle config anyways.